esx_service/cli/vmdkops_admin.py

```diff
@@ -419,7 +419,7 @@ def commands():
                     'required': True
                 },
                 '--tenant': {
-                    'help': 'Name of the tenant which volume to set belongs to.',
+                    'help': 'Name of the tenant the volume belongs to.',
```

Comment:
Please make sure to resolve the conflicts with the other change which renames tenant to vm-group.